pre_install_report/library/pre_install_check.py

```diff
@@ -76,21 +77,26 @@ def __init__(self, sas_logger: ViyaARKLogger, viya_kubelet_version_min,
 
     def _parse_release_info(self, release_info):
         """
-        This method checks that the format of the VIYA_KUBELET_VERSION_MIN specfied in the
-        user modifiable properies file is valid.
+        This method checks that the format of the VIYA_K8s_VERSION_MIN specified in the
```

Comment:
Nit.   Typo in variable name, lowercase 's'

Reply:
Done